lib/features/thread/presentation/widgets/suggestion_box_widget.dart

```diff
@@ -1,88 +1,183 @@
+                ),
+                const SizedBox(
+                  width: 4,
+                ),
+                Text(
+                  title,
+                  maxLines: 1,
+                  overflow: TextOverflow.ellipsis,
+                  style: const TextStyle(
+                      fontSize: 13,
+                      color: AppColor.colorTextButtonHeaderThread),
+                ),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
   }
 
   Widget build() {
-    return Material(
-      elevation: _elevation ?? 3,
-      color: Colors.white,
-      child: ConstrainedBox(
-        constraints: BoxConstraints(
-          maxHeight: _suggestionHeight ?? 60,
+    return Container(
+      decoration: BoxDecoration(
+          boxShadow: [
+            BoxShadow(
+              color: Colors.grey.withOpacity(0.3),
+              spreadRadius: 0.5,
+              blurRadius: 2,
+              offset: const Offset(0, 0),
+            ),
+          ],
+          color: Colors.white,
+          borderRadius: const BorderRadius.all(
+            Radius.circular(20),
+          )),
+      child: SingleChildScrollView(
+        child: Padding(
+          padding: const EdgeInsets.fromLTRB(16, 16, 16, 0),
+          child: SizedBox(
+            height: (_listData.length * 48) + 64,
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Row(
+                  children: [
+                    suggestionFilterItem(
+                        AppLocalizations.of(_context).has_attachment,
+                        _imagePaths.icAttachmentsFilter,
+                        SearchFilter.HAS_ATTACHMENT),
+                    suggestionFilterItem(
+                        AppLocalizations.of(_context).last_seven_days,
+                        _imagePaths.icCalendarFilter,
+                        SearchFilter.LAST_SEVEN_DAYS),
+                    suggestionFilterItem(
+                        AppLocalizations.of(_context).from_me,
+                        _imagePaths.icProfileFilter,
+                        SearchFilter.FROM_ME),
+                  ],
+                ),
+                const SizedBox(
+                  height: 8,
+                ),
+                Text(
+                  AppLocalizations.of(_context).recent,
+                  maxLines: 1,
+                  overflow: TextOverflow.ellipsis,
+                  style: const TextStyle(
+                      fontSize: 13,
+                      color: AppColor.colorTextButtonHeaderThread),
+                ),
+                const SizedBox(
+                  height: 8,
+                ),
+                Expanded(
+                    child: Column(
+                        mainAxisAlignment: MainAxisAlignment.start,
+                        children: _listData.map((e) {
+                          return recentSearchItem(e);
+                        }).toList())),
```

Comment:
why not use `ListView.builder`

Reply:
I wrap column and other stuff in singleChildScrollView => it will scroll the whole content. However, if I use list view I only scroll the list of the recent word, not the whole content.

Reply:
it is not a good practice here, listView is better for the performance